examples/nlp/dialogue/dialogue.py

```diff
@@ -110,7 +113,7 @@ def main(cfg: DictConfig) -> None:
             model = model_class.from_pretrained(cfg.pretrained_model)
         else:
             logging.info(f'Restoring model from {cfg.model.nemo_path}')
-            model = model_class.restore_from(cfg.model.nemo_path)
+            model = model_class.restore_from(cfg.model.nemo_path, override_config_path=cfg.model, strict=False)
```

Comment:
Can you leave a comment above this saying why we need to override and why strict = False? It should be pretty straightforward e.g. another data_dir can be used and that some keys (slot_mlp and intent_mlp might not be found due to the zero shot transfer issue). Again, is zero shot important for you (i.e. does it perform well enough to be useful)?